setup.py

```diff
@@ -58,7 +58,7 @@ def read(fname):
         'importlib_metadata',  # drop when pycalphad drops support for Python<3.8
         'matplotlib>=3.3',
         'numpy>=1.13',
-        'pyparsing>=2.0,<3',
```

Comment:
This one is `2.0` while the other one is `2`. Do we need to set `2.4` as the minimum per your comment?

Reply:
At minimum we need 2.1 for the `stopOn` keyword argument, per https://github.com/pyparsing/pyparsing/blob/d108a29db062c9250f50c978e3a86381d1b0746b/CHANGES#L1552

Since we aren't backtesting pyparsing extensively and there do seem to be subtle differences between versions, pinning to `pyparsing>=2.4` is probably the best move.

Reply:
Sounds good to me